condition.go

```diff
@@ -117,3 +120,57 @@ func Lte(column string, value interface{}) Builder {
 		return buildCmp(d, buf, "<=", column, value)
 	})
 }
+
+// Lk is `LIKE`.
+// When value is nil, do nothing.
+// When value is a slice, do nothing.
+// Otherwise it will be translated to `LIKE`.
+func Like(column string, value interface{}) Builder {
+	return BuildFunc(func(d Dialect, buf Buffer) error {
+		if value == nil {
+			return fmt.Errorf("Column %s LIKE nil is Not Supported", column)
+		}
+
+		v := reflect.ValueOf(value)
+		switch {
+		case v.Kind() == reflect.TypeOf([]rune{}).Kind() && v.Len() != 0:
+			return buildCmp(d, buf, "LIKE", column, string(value.([]rune)))
+
+		case v.Kind() == reflect.TypeOf([0]rune{}).Kind() && v.Len() != 0:
+			return buildCmp(d, buf, "LIKE", column, string(value.([]rune)[:]))
+
+		case v.Kind() == reflect.Slice || v.Kind() == reflect.Array:
+			return fmt.Errorf("Column %s LIKE Multiple Value is Not Supported", column)
+
+		default:
+			return buildCmp(d, buf, "LIKE", column, value)
+		}
+	})
+}
+
+// Nlk is `NOT LIKE`.
+// When value is nil, do nothing.
+// When value is a slice, do nothing.
+// Otherwise it will be translated to `NOT LIKE`.
+func NotLike(column string, value interface{}) Builder {
+	return BuildFunc(func(d Dialect, buf Buffer) error {
+		if value == nil {
+			return fmt.Errorf("Column %s NOT LIKE nil is Not Supported", column)
+		}
+
+		v := reflect.ValueOf(value)
+		switch {
+		case v.Kind() == reflect.TypeOf([]rune{}).Kind() && v.Len() != 0:
```

Comment:
I think it is not needed to check that v.Len() != 0, becuase 'LIKE ""' is valid expression.